source/dlangbot/app.d

```diff
@@ -164,17 +164,19 @@ void handlePR(string action, PullRequest* _pr)
 
     if (action == "labeled" || action == "synchronize")
     {
-        auto labelsAndCommits = handleGithubLabel(pr);
+        auto labels = pr.labels;
         if (action == "labeled")
+        {
+            if (auto method = labels.autoMergeMethod)
+                commits = pr.tryMerge(method);
             return;
+        }
         if (action == "synchronize")
```

Comment:
Looks a bit as if `action == "synchronize"` was added later, anyhow thanks for the fix.